src/reducers/utils.js

```diff
@@ -112,24 +95,24 @@ function isNonStandardNonrepeatableSubfield(tag, subfieldCode) {
 
 
 export function subfieldIsRepeatable(tag, subfieldCode) {
-  const fieldSpecs = melindaFields.fields.filter(field => field.tag === tag);
-  if (fieldSpecs.length !== 1) {
-    nvdebug(` WARNING! Getting field ${tag} data failed! ${fieldSpecs.length} hits. Default value true is used for'${subfieldCode}' .`, debugDev);
-    return true;
-  }
-
 
   if (isNonStandardNonrepeatableSubfield(tag, subfieldCode)) {
     return false;
   }
 
-
   // These we know or "know":
+  // NB! $9 should really be non-repeatable (However, MET-300) and allowed in only certain tags (do fixer/validator for that?)....
```

Comment:
$5 is non-repeatable and not usable in all fields according to MARC 21 format - but we use it locally as repeatable and usable in (almost) all fields. 
$9 is repeatable and usable in all fields.